src/python/foglamp/storage/payload_builder.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Storage layer python client
+"""
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+import urllib.parse
+from collections import OrderedDict
+from foglamp import logger
+
+_LOGGER = logger.setup(__name__)
+
+
+class PayloadBuilder(object):
+    """ Payload Builder to be used in Python wrapper class for Storage Service
+    Ref: https://docs.google.com/document/d/1qGIswveF9p2MmAOw_W1oXpo_aFUJd3bXBkW563E16g0/edit#
+    Ref: http://json-schema.org/
+    """
+
+    def __init__(self, payload=OrderedDict()):
+        self.payload = payload
+
+    @staticmethod
+    def verify_condition(arg):
+        retval = False
+        if isinstance(arg, list):
+            if len(arg) == 3:
+                if arg[1] in ['<', '>', '=', '>=', '<=', 'LIKE', 'IN', '!=']:
```

Comment:
LIKE and IN are not currently supported by the storage service, these can be added if we need them